test/interface/inplace_interpolation.jl

```diff
@@ -25,7 +25,7 @@ out_VMF = vecarrzero(ntt, size(prob_ode_2Dlinear.u0))   # Vector{Matrix{Float64}
         @test_throws MethodError sol_ODE(out_VF, tt; idxs = 1:1)
         @test sol_ODE(out_VF, tt) isa Vector{Float64}
         @test sol_ODE(out_VVF_1, tt) isa Vector{Vector{Float64}}
-        @test sol_ODE_interp.u == out_VF
+        @test sol_ODE_interp.u ≈ out_VF
```

Comment:
why would this only end up approximate?

Reply:
That may actually have been because of the missing `dv` in `rosenbrock_interpolants` which caused hermite to be used when it shouldn't.

Reply:
So it was just a temporary thing that should be fixed?

Reply:
```suggestion
        @test sol_ODE_interp.u == out_VF
```